### PR TITLE
ショッピングカートから商品を削除する機能を作成しました

### PR DESCRIPTION
**Pull request by SekiMoemi**
ご確認お願いします。

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package jp.co.example.ecommerce_a.controller;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:1]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.PackageNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name 'jp.co.example.ecommerce_a.controller' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import jp.co.example.ecommerce_a.service.DeleteShoppingCartService;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:8]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'jp.co.example.ecommerce_a.service.DeleteShoppingCartService'

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:8]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'jp.co.example.ecommerce_a.service.DeleteShoppingCartService' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * ショッピングカートから削除する情報を扱うコントローラ.
+ * 
+ * @author moemi.seki
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@author' should be preceded with an empty line.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ */
+@Controller
+@RequestMapping("/deleteShoppingCart")
+public class DeleteShoppingCartController {
+	@Autowired
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	private DeleteShoppingCartService deleteShoppingCartService;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+	/**
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:22]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+	 * ショッピングカートから商品を削除します.
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:23]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+	 *
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:24]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+@Controller
+@RequestMapping("/deleteShoppingCart")
+public class DeleteShoppingCartController {
+	@Autowired
+	private DeleteShoppingCartService deleteShoppingCartService;
+
+	/**
+	 * ショッピングカートから商品を削除します.
+	 * 
+	 * @param orderItemId 削除する注文商品のID
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+	 * @return ショッピングカート画面
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:26]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+	 */
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+	@GetMapping("")
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+	public String deleteShoppingCart(Integer orderItemId) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:29]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+		if (orderItemId == null) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:30]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:30]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+			return "redirect:/showShoppingCart";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:31]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:31]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+		}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+		deleteShoppingCartService.deleteShoppingCart(orderItemId);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:33]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:33]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+		return "redirect:/showShoppingCart";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:34]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:34]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+	}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:35]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:35]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java [hunk 20]
```diff
@@ -36,4 +37,15 @@ public Integer save(OrderItem orderItem) {
 		template.update(sql.toString(), param, keyHolder, keyColumnNames);
 		return keyHolder.getKey().intValue();
 	}
+
+	/**
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by SekiMoemi — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|42 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L42) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|43 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L43) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|44|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L44) Javadoc tag '@param' should be preceded with an empty line.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|44 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L44) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|45 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L45) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|46 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L46) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|46 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L46) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|47 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L47) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|47 col 17|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L47) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|48 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L48) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|48 col 17|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L48) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|49 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L49) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|49 col 17|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L49) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|50 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L50) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java|50 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/repository/OrderItemRepository.java#L50) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|1 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L1) Package name 'jp.co.example.ecommerce_a.service' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|6 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L6) Extra separation in import group before 'jp.co.example.ecommerce_a.repository.OrderItemRepository'
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|6 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L6) Wrong lexicographical order for 'jp.co.example.ecommerce_a.repository.OrderItemRepository' import. Should be before 'org.springframework.stereotype.Service'.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|11|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L11) Javadoc tag '@author' should be preceded with an empty line.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|16 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L16) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|16 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L16) 'member def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|17 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L17) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|19 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L19) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|20 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L20) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|21 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L21) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|22|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L22) Javadoc tag '@param' should be preceded with an empty line.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|22 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L22) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|23 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L23) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|24 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L24) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|24 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L24) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|25 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L25) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|25 col 17|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L25) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|26 col 1|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L26) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java|26 col 9|](http://github.com/ecsite-project/ec-202304a/blob/0de80d8239d5e12e2d233094aed5026bee668086/src/main/java/jp/co/example/ecommerce_a/service/DeleteShoppingCartService.java#L26) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
</details>

**Review by yamada-kensyu — APPROVED** [review]
承認します

**Pull request merged**